contrib/caldav/calcurse-caldav.py

```diff
@@ -674,7 +672,14 @@ def run_hook(name):
 sync_filter = config.get('General', 'SyncFilter')
 verbose = verbose or config.get('General', 'Verbose')
 
-password = password or config.get('Auth', 'Password')
+if os.getenv('CALCURSE_CALDAV_PASSWORD'):
+    # This approach is deprecated, but preserved for backwards compatibility
+    password = os.getenv('CALCURSE_CALDAV_PASSWORD')
+elif config.get('Auth', 'Password'):
+    password = config.get('Auth', 'Password')
+elif config.get('Auth', 'PasswordCommand'):
+    password = subprocess.run(config.get('Auth', 'PasswordCommand').split(' ', 1), capture_output=True).stdout.decode('UTF-8')
```

Comment:
This only work correctly in corner cases. Could you please change this to tokenize the command with shlex.split()? Please also ensure the password variable gets initialized if none of the three cases apply (i.e., no password is provided).

Everything else LGTM, thanks for your contribution and sorry for the delayed review!

Reply:
Thanks so much for your feedback; the latest changes have been applied in a force-push. :pray: